tsconfig.json

```diff
@@ -1,17 +1,18 @@
 {
   "compilerOptions": {
-    "moduleResolution": "Node",
     "declaration": true,
+    "esModuleInterop": true,
+    "lib": [
+      "ES2017"
+    ],
     "module": "ES6",
+    "moduleResolution": "Node",
+    "outDir": "dist",
```

Comment:
I'd suggest specifying a `rootDir` of `./src`, to ensure that we don't accidentally include a module outside of dir and mess up the resulting directory structure of the `tsc` build.